blocks/research/research.css

```diff
@@ -0,0 +1,17 @@
+.research {
+  display: flex;
+  background: rgb(242, 242, 242);
+  padding: 40px 130px;
+  gap: 73px;
+}
+@media screen and (max-width:1199px) {
```

Comment:
@Woolfs1423 At the same time, let's add one new line to separate different blocks for better readability. 

```css
.research {
  display: flex;
  background: RGB(242, 242, 242);
  padding: 40px 130px;
  gap: 73px;
}

@media screen and (max-width:1199px) {
  .research {
    background: RGB(255, 255, 255);
    padding: 54px 20px 44px ;
  }
}

@media screen and (max-width: 659px) {
  .research {
    display: none;
  }
}
```

Please double-check all your CSS file on that